website/cue/reference/components/sinks/console.cue

```diff
@@ -21,7 +21,7 @@ components: sinks: console: {
 				enabled: true
 				codec: {
 					enabled: true
-					enum: ["json", "text"]
+					enum: ["json", "text", "ndjson"]
```

Comment:
How does json and ndjson differ for the console sink?

Reply:
This one is a bit odd. Technically `json` encoding doesn't have a newline at the end, but the `console` sink adds one anyway for readability. The `ndjson` adds a newline as part of the encoding, and then a 2nd one is added by the sink.

We should probably remove the newline added by the sink so you can choose if you actually want one or not with the encoding, but that would break backwards compatibility. I'm open to suggestions here if it's worth changing or not.

Reply:
🤔 that _feels_ like a reasonable breaking change to make. Maybe we could alias ndjson and json for a release before changing the behavior?

@jszwedko what do you think?

Reply:
I think that make sense. This may also change shortly with Pablo's work on adding similar framing/codec support to sinks that we have on sources though so we could opt to punt for now. I think, in the future, it will look like:

```
encoding.codec = "json"
framing.method = "newline_delimited"
```

If we do that, I'd just leave `ndjson` out of the docs for now even if the config would allow it.

Reply:
😆 I summon @pablosichert for his thoughts too then

Reply:
I'm going to remove `ndjson` from the docs for now, and we can revisit once the Pablo's work is done.